### DISCUSSION TITLE
Gas vents drain all tanks in Exosuit

### DISCUSSION THREAD

**Discussion by ViviX12** [FakeFishGames/Barotrauma#14097] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If there are multiple oxygen tanks in a worn exosuit, swimming through a cave gas vent will drain all of them, instead of just the active one.

### Reproduction steps

1. Store multiple oxygen tanks in an exosuit cargo slots
2. Swim through a cave vent

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ViviX12** [comment]
Soon after also noticed tanks in unequipped oxygen masks are also drained. Tanks anywhere else in teh inventory are not affected, so I'm assuming it's not an intended behaviour.

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/10423
